eventually/templates/emails/registration.html

```diff
@@ -2,3 +2,4 @@ <h1>Welcome <i>{{ first_name }}</i>!</h1>
 
 <p>Congratulations, your account almost created</p>
 <p>Please click on the link to <a href="http://{{domain}}/api/v1/user/activate/{{token}}">activate your account</a>.</p>
+<p>You have {{time_left}}left to confirm<p>
```

Comment:
add space after `{{time_left}}` and add the newline to the end of the file

Reply:
done